scripts/client_test.sh

```diff
@@ -57,9 +57,31 @@ elif [[ "$JOB_TYPE" == "predict" ]]; then
       --minibatch_size=64 \
       --num_minibatches_per_task=2 \
       --num_workers=2 \
-      --job_name=test-mnist-predict \
+      --job_name=test-predict \
       --log_level=INFO \
       --image_pull_policy=Never
+elif [[ "$JOB_TYPE" == "odps" ]]; then
+    elasticdl train \
+      --image_base=elasticdl:ci \
+      --model_zoo=model_zoo \
+      --model_def=odps_iris_dnn_model.odps_iris_dnn_model.custom_model \
+      --training_data_dir=$ODPS_TABLE_NAME \
```

Comment:
This parameter should be renamed, right?

Reply:
Yes, but probably need more people's feedback on https://github.com/sql-machine-learning/elasticdl/issues/1248 before renaming those parameters